cmd/main.go

```diff
@@ -4,32 +4,55 @@
  **********************************************************************/
 package main
 
+import "C"
 import (
 	"os"
 	"rpc"
 	"rpc/internal/amt"
 	"rpc/internal/client"
 	"rpc/internal/rps"
+	"rpc/pkg/utils"
+	"strings"
 
 	log "github.com/sirupsen/logrus"
 )
 
-func runRPC(args []string) {
+func checkAccess() (int, error) {
```

Comment:
Why are we moving everything from lib.go into main.go?  What was wrong with it being the way it was before?  Seems out of scope for the user story

Reply:
methods were redundant duplication. why was there a cmd/lib.go to create a user library in the cmd namespace in the first place. which calls back into the main.go file anyways? and you don't 'execute' a library, you link to it. so not sure how that ended up in the cmd/ folder to start with?

Reply:
The reason is for avoiding gcc requirement on either OS if not using as a library. Please put it back.

Reply:
done, tested executable and sample on Ubuntu, 
tested  only executable on Win ... no gcc installed yet